squeemishly/cloney-island
# makes minor changes to styling on nav bar

#### What does  this PR do?
Makes minor changes fixing some issues with styling and responsiveness. 
#### Where should the reviewer start?
nav-bar.scss
#### How should this be manually tested?
run
`rails s`
go to localhost:3000
navigate through fledged out pages to make sure logo loads. 
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Questions:
  - Do Migrations Need to be ran? 
  - Do Environment Variables need to be set? 
  - Any other deploy steps?
